mozilla/blurts-server
# Avoid people thinking 14 days fixes *all* scan results

# References:

Jira: MNTOR-3200
Figma: https://www.figma.com/design/DTbmXzCQCw2PxXpHQW8yfW/Monitor-MVP-Enhancements?node-id=1926-1291&t=yj8SkckZSeRi6GFg-4



# Description

We've had a number of people reach out, surprised that some data removals took more than 14 days. Thus, this removes mention of that time period.

# How to test

Basically, there should be no more mentions of removals typically taking 7-14 days, but for the exact strings, see Figma. The changes are in these places:

- [Welcome to Monitor Plus](https://fx-monitor-storybook.netlify.app/?path=/story/pages-logged-in-guided-resolution-1e-welcome-to-plus--welcome-to-plus-view-in-progress-story), with scan results.
- [About exposure types dialog](https://fx-monitor-storybook.netlify.app/?path=/story/pages-logged-in-dashboard--dashboard-us-no-premium-unresolved-scan-unresolved-breaches) - to reach this, open the dashboard with your screen wide enough so the column headings above the exposure cards are visible, then click the "(?)" next to "Exposure type".
- [About what we fixed dialog](https://fx-monitor-storybook.netlify.app/?path=/story/pages-logged-in-guided-resolution-2-high-risk-data-breaches--high-risk-breach-done-story) - to reach this, visit the "Done" step for any of the guided flow flows, then click the "(?)" in the box with the three numbers. Note that the Figma shows different text that doesn't align with the values in the box, so I've just updated it with my own suggestion ~~and [asked on Slack](https://mozilla.slack.com/archives/CEV7MGSQH/p1721737434258949) for the correct text~~ which was OK'ed.
- "How long do removals take?" dialog - I could not find this dialog. ~~[Asked on Slack.](https://mozilla.slack.com/archives/CEV7MGSQH/p1721737550752899?thread_ts=1721737434.258949&cid=CEV7MGSQH)~~ Confirmed that no action is needed there.

# Checklist (Definition of Done)

- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, just string changes
